doc/adr/0012-commit-to-datatypes-in-cardano-api.rst

```diff
@@ -0,0 +1,151 @@
+.. _commit-to-datatypes-in-cardano-api:
+
+ADR 12: Commit to datatypes in cardano-api
+==========================================
+
+Date: 2022-10-03
+
+Author(s)
+---------
+
+koslambrou <konstantinos.lambrou@iohk.io>
+
+Status
+------
+
+Draft
+
+Context
+-------
+
+Since the genesis of the ``plutus-apps`` repository, the components have been
+historically using the types in the ``plutus-ledger-api`` package (which is now
+part of ``plutus`` repository) in the off-chain part of a Plutus application.
+
+This was desirable in order to start designing a way to build Plutus application before the ``cardano-ledger`` actually supported the Alonzo-era features.
+Of course, this resulted in the unintented consequence that we used ``TxInfo`` types (types that are designed to be used in Plutus scripts) in off-chain code.
+This wouldn't has been a problem if there was 1:1 relationship between on-chain and off-chain types.
```

Comment:
"has been" should be "have been";
suggest inserting "a" --> if there was a 1:1 relationship...